Userland/Utilities/headless-browser.cpp

```diff
@@ -379,6 +418,11 @@ static ErrorOr<int> run_tests(HeadlessWebContentView& view, StringView test_root
         else
             outln("");
 
+        if (test.result.has_value() && test.result.value() == TestResult::Skipped) {
```

Comment:
Does it make more sense to check `s_skipped_tests` here? Instead of duplicating the check twice above in the `collect_*_tests` methods + needing a special check here anyways.